DESCRIPTION

```diff
@@ -36,6 +36,7 @@ Imports:
     ggplot2,
     glue,
     hardhat (>= 1.3.0),
+    here,
```

Comment:
```suggestion
```

This isn't necessary except for ease in local vignette rendering (possibly). See below.